### DISCUSSION TITLE
Run only dotnet tests of special Category using cmd

### DISCUSSION THREAD

**Discussion by goldlife02** [nunit/nunit#4614] [Q&A]
I know `CategoryAttribute` is used to run only special tests. But I don't know how to execute the functions which have a `CategoryAttribute`.
For example, to execute the tests which have `[Category("PlaywrightTest")]`, which command line (starts with `dotnet test`) should I use?
Thank you.

**Comment by mikkelbu** [comment]
I think blogpost explains it better than I can https://blog.prokrams.com/2019/12/16/nunit3-filter-dotnet/
